packages/core/contracts-ovm/insured-bridge/external/AVM_CrossDomainEnabled.sol

```diff
@@ -9,51 +7,9 @@ abstract contract AVM_CrossDomainEnabled {
     }
 
     uint160 constant offset = uint160(0x1111000000000000000000000000000000001111);
-    iAVM_Inbox public immutable inbox;
```

Comment:
I moved any logic only needed for L1 to "Arbitrum_CrossDomainEnabled" and relocated to `/contracts`